wetmap/src/components/newModals/setting/view.tsx

```diff
@@ -0,0 +1,28 @@
+import screenData from '../screenData.json';
+import React from 'react';
+import styles from './style.module.scss';
+import ButtonIcon from '../../reusables/buttonIcon';
+import Icon from '../../../icons/Icon';
+
+type SettingsProps = {
+  onClose:      () => void
+};
+
+export default function SettingsView(props: SettingsProps) {
+  return (
+    <>
+      <div className={styles.buttonBack}>
+        <ButtonIcon
+          icon={<Icon name="chevron-left" />}
+          className="btn-lg text-gray"
+          onClick={(e) => {
+            if (props.onClose) {
+              props.onClose(e);
+            }
+          }}
+        />
+        <h1 className="ml-4 mt-4 text-bold">{screenData.SettingsPage.header}</h1>
```

Comment:
minor style request can you please bump the ml-4 to ml-10, I think that looks a bit nicer :)

Reply:
Just pushed this change to web-46.